src/main/java/com/inf2b/projectcodegeneration/controllers/TransactionController.java

```diff
@@ -0,0 +1,23 @@
+package com.inf2b.projectcodegeneration.controllers;
+
+import com.inf2b.projectcodegeneration.models.Transaction;
+import com.inf2b.projectcodegeneration.services.TransactionService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/transactions")
+public class TransactionController {
+
+    private final TransactionService transactionService;
+
+    public TransactionController(TransactionService transactionService){
+        this.transactionService = transactionService;
+    }
+
+    @PostMapping("/transactions")
```

Comment:
```suggestion
    @PostMapping
```
Je hebt boven de class name al de path /transaction gedefinieerd, als je deze neerzet dan is het dus /transaction/transaction